app/views/shared/provider/_header.html.slim

```diff
@@ -1,53 +1,42 @@
+= javascript_pack_tag 'header'
```

Comment:
I think it's better to have a pack for each module, since we are merging existing navigation into a header, why not using the same pack?